Product/Production/Adapters/General/CONNECTAdminGUI/src/main/java/gov/hhs/fha/nhinc/admingui/services/GatewayService.java

```diff
@@ -139,7 +139,7 @@ public boolean discoverPatient(PatientSearchBean patientQuerySearch) {
             populatePatientBean(patientDiscoveryResults, patientQuerySearch);
             return true;
         } catch (Exception ex) {
-            LOG.error("Failed to Retrieve Patient Data" + ex.getMessage());
+            LOG.error("Failed to Retrieve Patient Data: {}", ex.getLocalizedMessage(), ex);
             //TODO: notify the UI or somehow inform the user
```

Comment:
This TODO was added for a reason, better to deal with it in another ticket than to remove it just so we can get one less violation count.